src/App/App.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import './App.css';
+import AllMovies from '../AllMovies/AllMovies';
+import TopMovies from '../TopMovies/TopMovies'
+import Nav from '../Nav/Nav';
+import CurrentMovie from '../CurrentMovie/CurrentMovie';
+import {
+  getAllMovies, 
+  getSingleMovie 
+} from '../apiCalls'
+
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      movies: [],
+      currentMovie: '',
```

Comment:
Typically, the datatype of a state value does not change. Here, it's going from a `string` to an `object`. However, and I think this is your case, I have seen a piece of state starting out at `null` as a signal to other developers that the information has not been set yet. So in this case, I would use `null` instead of the empty string.